Failure TestSchedulerScheduleOne

### Failure cluster [ec3c8dc0b952934cd3b8](https://go.k8s.io/triage#ec3c8dc0b952934cd3b8)

##### Error text:
```
Failed
=== RUN   TestSchedulerScheduleOne/prebind_failed_with_status_code_error_(Queueing_hints_enabled:_true,_Async_API_calls_enabled:_false,_NominatedNodeNameForExpectation_enabled:_true)
    schedule_one_test.go:1084: Expected nominated node name "node1", got ""
--- FAIL: TestSchedulerScheduleOne/prebind_failed_with_status_code_error_(Queueing_hints_enabled:_true,_Async_API_calls_enabled:_false,_NominatedNodeNameForExpectation_enabled:_true) (0.30s)

=== RUN   TestSchedulerScheduleOne
--- FAIL: TestSchedulerScheduleOne (18.55s)

```
#### Recent failures:
[7/30/2025, 5:39:28 AM ci-kubernetes-unit](https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/ci-kubernetes-unit/1950400728660971520)
[7/30/2025, 4:50:10 AM ci-kubernetes-unit-windows-master](https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/ci-kubernetes-unit-windows-master/1950388397763727360)
[7/30/2025, 4:10:08 AM ci-kubernetes-unit-ppc64le](https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/ci-kubernetes-unit-ppc64le/1950376820561940480)
[7/30/2025, 2:06:57 AM ci-kubernetes-unit-ppc64le](https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/ci-kubernetes-unit-ppc64le/1950346621065629696)
[7/30/2025, 1:50:07 AM ci-kubernetes-unit-windows-master](https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/ci-kubernetes-unit-windows-master/1950343098320031744)


/kind failing-test

<img width="1019" height="682" alt="Image" src="https://github.com/user-attachments/assets/23845130-66a9-402b-bb0c-1a61987c455e" />

<!-- If this is a flake, please add: /kind flake -->

<!-- Please assign a SIG using: /sig SIG-NAME -->

--- a/pkg/scheduler/schedule_one_test.go
+++ b/pkg/scheduler/schedule_one_test.go
@@ -1064,7 +1064,7 @@ func TestSchedulerScheduleOne(t *testing.T) {
 							mu.Lock()
 							updatedNominatedNodeName = updatedPod.Status.NominatedNodeName
 							mu.Unlock()
-							return true, nil, nil
+							return false, nil, nil
 						})
 
 						schedFramework, err := tf.NewFramework(ctx,

